lib/data_update_scripts/20210630034523_add_work_profile_field.rb

```diff
@@ -0,0 +1,12 @@
+module DataUpdateScripts
+  class AddWorkProfileField
+    def run
+      ProfileField.find_or_create_by(
+        label: "Work",
+        placeholder_text: "What do you do? Example: Software Engineer at Forem",
```

Comment:
should the place holder be Forem specific?

Reply:
I hear you but couldn't come up with a  good generic example. Suggestions?